packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -842,15 +842,68 @@ class SlideshowView extends GalleryComponent {
     return <GalleryDebugMessage {...this.props.debug} />;
   }
 
+  getArrowsRenderData() {
+    const { customNavArrowsRenderer, arrowsSize } = this.props;
+    if (customNavArrowsRenderer) {
+      return {
+        arrowRenderer: customNavArrowsRenderer,
+        navArrowsContainerWidth: arrowsSize,
+        navArrowsContainerHeight: arrowsSize
+      }
+    }
+
+    const arrowOrigWidth = 23; //arrow-right svg and arrow-left svg width
+    const arrowOrigHeight = 39; //arrow-right svg and arrow-left svg height
+    const scalePercentage = arrowsSize / arrowOrigWidth;
+    const svgStyle = { transform: `scale(${scalePercentage})` };
+
+    const navArrowsContainerWidth = arrowsSize; // === arrowOrigWidth * scalePercentage
+    const navArrowsContainerHeight = arrowOrigHeight * scalePercentage;
+
+    const svgInternalStyle = {};
+    if (utils.isMobile()) {
+      const { arrowsColor } = this.props;
+      if (typeof arrowsColor !== 'undefined') {
+        svgInternalStyle.fill = arrowsColor.value;
+      }
+    }
+
+    const arrowRenderer = (position) => {
+      if (position === 'left') {
+        return (
+          <svg width="23" height="39" viewBox="0 0 23 39" style={svgStyle}>
```

Comment:
I think it's better to reuse the variables you defined earlier:
```
          <svg width={arrowOrigWidth} height={arrowOrigHeight} viewBox={`0 0 {arrowOrigWidth} {arrowOrigHeight`} style={svgStyle}>
```

Reply:
Yes I see, but the reason those variables were defined as they were (23, 39) is because the SVG itself, as the SVG is sort of a "hardcoded" UI component.

Reply:
Ok, but we can make it less hardcoded with the constant defined in the same function. It will clean it up a bit :)